docker-hub/repos.md

```diff
@@ -36,6 +38,11 @@ When creating a new repository:
 After you hit the **Create** button, you can start using `docker push` to push
 images to this repository.
 
+
+## Deleting a repository
+
+ Sign into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} with your user account to delete a repository. Navigate to the repository **Settings** and click **Delete Repository** to delete the repository and all images stored within it.
```

Comment:
I think the order is:
1. Sign into Docker Hub.
2. Click Repositories.
3. Select a repository from the list and click **Settings** and then **Delete Repository**. Deleting a repository deletes all the images it contains and its build settings. This action cannot be undone.
4. Enter the name of the repository to confirm the deletion.
5. Click **Delete**.

Reply:
Updated